vendor/engines/secure_rooms/app/models/secure_room_price_policy.rb

```diff
@@ -2,4 +2,22 @@ class SecureRoomPricePolicy < PricePolicy
 
   include PricePolicies::Usage
 
+  def estimate_cost_and_subsidy_from_order_detail(order_detail)
+    calculate_cost_and_subsidy_from_order_detail(order_detail)
+  end
+
+  def calculate_cost_and_subsidy_from_order_detail(order_detail)
+    entry_at = order_detail.occupancy.entry_at
```

Comment:
What's the plan for determining whether or not we may need a `try`? I gather there's a lot of pieces still to come with SecureRooms and pricing--I just want to make sure we don't lose track of this and ship something to prod with a weird edge case error buried in the code.